src/archivist/services/sourceDocument.js

```diff
@@ -35,6 +35,11 @@ export default class SourceDocument {
     return result.filter(selector => selector);
   }
 
+  clearContent() {
+    this.content = null;
```

Comment:
> You know better, but just sharing that it might be useful to set some sort of indicator in the object that content has been cleared, so that empty content is not confused with an issue in the content loading itself… ?